src/dynamicprompts/commands/variant_command.py

```diff
@@ -21,15 +22,18 @@ class VariantOption:
 class VariantCommand(Command):
     variants: list[VariantOption]
     min_bound: int = 1
-    max_bound: int = 1
+    max_bound: int = None
     separator: str = ","
     sampling_method: SamplingMethod | None = None
 
     def __post_init__(self):
-        min_bound, max_bound = sorted((self.min_bound, self.max_bound))
+        min_bound = self.min_bound
+        if self.max_bound:
```

Comment:
```suggestion
        if self.max_bound is not None:
```
`0` is _technically_ a valid `max_bound`.

Reply:
0 is also an invalid value for max

Reply:
It's not. It just means that up to zero things should be generated, which is perfectly valid.